### ISSUE TITLE
Breakpoints on web

### ISSUE THREAD

**Issue by heondo** [jaredh159/tailwind-react-native-classnames#153]
I cant seem to get the breakpoints `md:bg-green-500` to work in any capacity. Is this supported or in the roadmap?

**Comment by jaredh159** [comment]
yup, it's supported and it works. the most common error for this not working would be failing to properly connect the `tw` function to the source of runtime device context with `useDeviceContext()`, make sure you've carefully read and done all the steps in the readme, particularly around:

https://github.com/jaredh159/tailwind-react-native-classnames#enabling-device-context-prefixes

**Comment by heondo** [comment]
oh my, so sorry for missing that. thank you so much

**Comment by jaredh159** [comment]
no problem, glad you got it figured out
